### ISSUE TITLE
Screen reader legibility and formatting on Family and Caregiver Benefits page

### ISSUE THREAD

**Issue by Shelby-Arnett** [department-of-veterans-affairs/va.gov-cms#24065]
## Current Behavior
On Dev and Prod, the blue alert block on the [Family and Caregiver Benefits page](https://dev.va.gov/family-and-caregiver-benefits/) has one line of text that cannot be read smoothly by a screen reader and is broken up by the two apostrophes: "If you’re concerned about a Veteran who’s homeless or at risk of becoming homeless"

In addition, the same apostrophes don't have bold formatting like the rest of the sentence.

## Expected Behavior
This line of text should read like a normal sentence, without any awkward breaks. The apostrophes should also be bold, like the rest of the sentence.

## Reproduction Steps
1. Navigate to https://dev.va.gov/family-and-caregiver-benefits/
2. Click "Find out how to get support anytime, day or night" to open the alert box accordion.
3. Inspect the following line to see how the code is being broken up at each apostrophe: "If you’re concerned about a Veteran who’s homeless or at risk of becoming homeless"

**Next Build**
<img width="542" height="66" alt="Image" src="https://github.com/user-attachments/assets/6d020d36-2c1b-4de9-91d9-58bedc404924" />

**Comment by klwatanabe** [comment]
Related slack thread: https://dsva.slack.com/archives/C01SR56755H/p1777470987234339

This appears to be an issue with the content configuration in Drupal. Alex handed this off to the Content and and IA team.
